docker/images/kumascript/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.10.0-slim
+FROM node:10.14.2-slim
```

Comment:
I'd support ``node:10-slim``, to always get the latest minor version of node 10, but I'm not sure how often node breaks things in minor releases.